give.php

```diff
@@ -514,8 +514,10 @@ private function includes() {
 			/**
 			 * Blocks
 			 */
-			require_once GIVE_PLUGIN_DIR . 'blocks/donation-form/class-give-donation-form-block.php';
-			require_once GIVE_PLUGIN_DIR . 'blocks/donation-form-grid/class-give-donation-form-grid-block.php';
+			if ( function_exists( 'register_block_type' ) ) {
+				require_once GIVE_PLUGIN_DIR . 'blocks/donation-form/class-give-donation-form-block.php';
+				require_once GIVE_PLUGIN_DIR . 'blocks/donation-form-grid/class-give-donation-form-grid-block.php';
+			}
 
```

Comment:
@ajitbohra This will resolve fatal error caused by only the `register_block_type` function. In future we might use more functions that are defined inside Gutenberg. I think we should require Block-related files if Gutenberg is activated. What do you say?

Reply:
@Sidsector9  This will prevent including block registration files if Gutenberg is not active same as checking if the plugin is activated. I am sure we might have an alternative way to achieve the same.

@ravinderk your thoughts

Reply:
@ajitbohra I see your point, checking 1 function is enough to test if a plugin is active or not, in case if they remove the `register_block_type` function entirely or update the name at any point in the future, this would not be the most robust way. @ravinderk

Reply:
For clarity I would use `if ( is_plugin_active( 'gutenberg' ) ) { ... }`. I understand checking if the function exists achieves the same result, but our goal is to only require those files if the plugin is active, and `is_plugin_active()` does exactly that.

Reply:
@kevinwhoffman @Sidsector9 agrees makes sense using 

`if ( is_plugin_active( 'gutenberg' ) ) { ... }` being future proof :+1:

Reply:
@Sidsector9 closing this PR as not sure if the code I am pushing will be inline with current code step up or not, personally feels like spreading spaghetti to me :p better to be handled by someone internal :)

By the way, It's always gr8 learning experience following activity here.